webapp/src/app.tsx

```diff
@@ -61,6 +61,19 @@ const App = (): JSX.Element => {
         getSubscribeModalState(usePlugin.state).visibility,
     ]);
 
+    // Fetch the list of linked projects list
+    useEffect(() => {
+        if (usePlugin.isUserAccountConnected()) {
+            usePlugin.makeApiRequestWithCompletionStatus(
+                plugin_constants.pluginApiServiceConfigs.getAllLinkedProjectsList.apiServiceName,
+            );
+        }
+    }, [
+        getCreateTaskModalState(usePlugin.state).visibility,
+        getSubscribeModalState(usePlugin.state).visibility,
+        getRhsState(usePlugin.state).isSidebarOpen,
```

Comment:
will this not cause an issue of RHS not updating when we link a new project or unlink a project until these states change?

Reply:
yes, this has been fixed in later PR